concordium-consensus/src/Concordium/Scheduler/EnvironmentImplementation.hs

```diff
@@ -60,7 +61,7 @@ data NoLogSchedulerState (m :: DK.Type -> DK.Type) = NoLogSchedulerState {
   _ssNextIndex :: !TransactionIndex
   }
 
-mkInitialSS :: UpdatableBlockState m -> NoLogSchedulerState m
+mkInitialSS :: forall m . UpdatableBlockState m -> NoLogSchedulerState m
```

Comment:
I'm not sure what the purpose of adding this (and ScopedTypeVariables) is?

Reply:
I think I needed to specify a monad explicitly in the tests for whatever reason. When this is needed I strongly prefer being explicit about the type variables instead of relying on implicit ordering without an explicit forall.